moped-database/migrations/1676056649901_dedupe_subcomponents_table/up.sql

```diff
@@ -0,0 +1,53 @@
+-- copy component <> subcomponent relationships from public.moped_subcomponents
```

Comment:
😵 
we're going from a table that looks like this:

component_id | subcomponent_id | subcomponent_name
-- | -- | --
16 | 1 | Audible push button'
16 | 2 | Bicycle signal'
16 | 3 | Leading pedestrian interval'
17 | 4 | Audible push button'
17 | 5 | Bicycle signal'
17 | 6 | Leading pedestrian interval'
18 | 7 | Audible push button'
18 | 8 | Bicycle signal'
18 | 9 | Leading pedestrian interval'
7 | 10 | Raised pavement markers'
8 | 11 | Raised pavement markers'
9 | 12 | Raised pavement markers'
10 | 13 | Raised pavement markers'
11 | 14 | Raised pavement markers'
12 | 15 | Raised pavement markers'
6 | 16 | Raised pavement markers'
21 | 17 | Raised pavement markers'
22 | 18 | Raised pavement markers'

to a table that looks like this:


subcomponent_id | subcomponent_name
-- | --
1 | Audible push button'
2 | Bicycle signal'
3 | Leading pedestrian interval'
4 | Raised pavement markers'

Reply:
Woohoo! 🚀